src/Gwen/APIHandlers/SpotifyAPI/load_cookies.py

```diff
@@ -0,0 +1,168 @@
+import pickle
+import spotipy
+import cred
+from selenium import webdriver
+from selenium.webdriver.chrome.service import Service
+from selenium.webdriver.chrome.options import Options
+from spotipy.oauth2 import SpotifyOAuth
+import time
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.by import By
+from selenium.common.exceptions import TimeoutException
+
+class Spotify:
+
+
+    #chrome driver
+    webdriver_service = Service(".\chromedriver.exe")

```

Comment:
Maybe what we should do is put this Exe in a special folder, Add that file Location to a OS Enviornment Variable